state_formatters/dp_formatters.py

```diff
@@ -749,6 +790,15 @@ def el_formatter_dialog(dialog: Dict):
     ]
 
 
+def el_formatter_last_bot_dialog(dialog: Dict):
+    # Used by: entity_linking annotator
+    if len(dialog["bot_utterances"]):
+        entity_substr_list, entity_tags_list, context = prepare_el_input_last_bot(dialog)
+    else:
+        entity_substr_list, entity_tags_list, context = [""], [""], [""]
+    return [{"entity_substr": [entity_substr_list], "entity_tags": [entity_tags_list], "context": [context]}]
```

Comment:
in another PR I have already asked for that, so I have to ask you too))
As these are batches, please name them in multiple way, so `contexts` not `context1 (and in your formatters bellow)